app/templates/components/pluggable-drop-down.hbs

```diff
@@ -0,0 +1,9 @@
+<div {{action "toggleDisplayItems"}} >
```

Comment:
I'd argue that `composable-drop-down` is a better name, but I'm not fussy.
very generic and reusable, well done ! 👍 

since you're using `dropDownBody` as a slot name, I'd recommend `dropDownHeader` instead of `selectedDropDown`. 

Body and Header are a common combination, so people would expect it, and `selectedDropDown` doesn't sound like a slot name, it sounds like a property of the component